contact.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="description" content="Explore my personal portfolio of front-end development projects. Discover my work as a developer, including websites and web applications I've created.">
+    <meta name="author" content="Mikael Elias">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
```

Comment:
Precisa trocar aqui pra Contact
```suggestion
    <title>Contact</title>
```

Reply:
Feito